fuse/dl/lightning/pl_epoch_summary.py

```diff
@@ -22,19 +22,19 @@
 from fuse.utils.misc.misc import get_pretty_dataframe
 
 import pytorch_lightning as pl
-from pytorch_lightning import Callback
+from pytorch_lightning.callbacks import Checkpoint
 
 from pytorch_lightning.utilities.rank_zero import rank_zero_only
 import torch
 import pandas as pd
 
 
-class ModelEpochSummary(Callback):
+class ModelEpochSummary(Checkpoint):
```

Comment:
Why?

I looked on `Checkpoint` class documentation: Why do we want the trainer to recognize it as a checkpointing callback? what are the differences? (I'm curious and didn't find something insightful on google nor chatGPT lol)